server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/node/tasks/ConcurrentSearchTasksIT.java

```diff
@@ -110,7 +110,7 @@ public void testConcurrentSearchTaskTracking() {
             Map<Long, List<ThreadResourceInfo>> threadStats = getThreadStats(SearchAction.NAME + "[*]", taskInfo.getTaskId());
             // Concurrent search forks each slice of 5 segments to different thread (see please
             // https://github.com/apache/lucene/issues/12498)
-            assertEquals((int) Math.ceil(getSegmentCount(INDEX_NAME) / 5.0) + 1, threadStats.size());
+            assertEquals((int) Math.ceil(getSegmentCount(INDEX_NAME) / 5.0), threadStats.size());
```

Comment:
Changes related to task executor for concurrent search (the current thread does work as well)